Description: Remove implicit declaration warnings by adding
appropriate #includes, and fix a typo in a prototype.
Author: B. Watson <yahlcru@gmail.com>

--

diff -Naur lv-4.51.orig/src/file.h lv-4.51.orig.patched/src/file.h
--- lv-4.51.orig/src/file.h	2020-11-12 02:03:33.560002050 -0500
+++ lv-4.51.orig.patched/src/file.h	2020-11-12 02:04:06.555999050 -0500
@@ -172,7 +172,7 @@
 public inline int IobufGetc( iobuf_t *iobuf );
 public inline int IobufUngetc( int ch, iobuf_t *iobuf );
 public offset_t IobufFtell( iobuf_t *iobuf );
-public int IobufFseeko( iobuf_t *iobuf, offset_t off, int mode );
+public int IobufFseek( iobuf_t *iobuf, offset_t off, int mode );
 public int IobufFeof( iobuf_t *iobuf );
 #endif
 #define IobufPutc( a, b )	putc( a, (b)->iop )
diff -Naur lv-4.51.orig/src/guess.c lv-4.51.orig.patched/src/guess.c
--- lv-4.51.orig/src/guess.c	2003-11-12 22:08:19.000000000 -0500
+++ lv-4.51.orig.patched/src/guess.c	2020-11-12 02:04:22.621997590 -0500
@@ -21,6 +21,7 @@
  */
 
 #include <stdio.h>
+#include <string.h>
 
 #include <import.h>
 #include <decode.h>
diff -Naur lv-4.51.orig/src/guesslocale.c lv-4.51.orig.patched/src/guesslocale.c
--- lv-4.51.orig/src/guesslocale.c	2004-01-05 03:41:22.000000000 -0500
+++ lv-4.51.orig.patched/src/guesslocale.c	2020-11-12 02:04:49.294995165 -0500
@@ -20,6 +20,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
+#include <ctype.h>
+
 #ifdef HAVE_SETLOCALE
 
 #include <stdlib.h>
